security/run-tck.sh

```diff
@@ -211,7 +212,7 @@ then
     echo "Configuring WildFly for the Old TCK"
     pushd $TS_HOME/bin
     $ANT_HOME/bin/ant config.vi
```

Comment:
@darranl It seems the various uses of $ANT_HOME/bin is not necessary. If so we should clean that out having the mix of usage is confusing. That shouldn't block this though.